### PR TITLE
(PUP-7930) Update Array - Hash example

### PR DESCRIPTION
**Pull request by joshcooper**
Array - Hash removes [key, value] entries in the hash from the array, similar to how Array + Hash adds [key, value] entries like:

```
   [1,2,3] + {a => 10, b => 20}    # => [1,2,3, [a, 10], [b, 20]]
```

### CODE REVIEW

File: language/expressions.md [hunk 1]
```diff
@@ -378,7 +378,8 @@ Examples:
     # LHS must evaluate to an Array or a Hash (or it is a form of arithmetic expression)
     [1,2,3,4,5,6] - [4,5,6]         # => [1,2,3]
     [1,2,3] - 3                     # => [1,2]
-    [1,2,b] - {a => 1, b => 20}     # => [2]
+    [1,2,b] - {a => 1, b => 20}     # => [1, 2, b]
+    [1,2[b,20]] - {b => 20}         # => [1, 2]
```
Comment (hlindberg) [inline language/expressions.md:382]:
```suggestion
    [1,2,[b,20]] - {b => 20}         # => [1, 2]
```

**Review by hlindberg — CHANGES_REQUESTED** [review]
typo

**Review by lucywyman — APPROVED** [review]
Approved with Henrik's suggestion!

File: language/expressions.md (hunk 1 above)
Comment (joshcooper) [reply language/expressions.md:382]: 🤦  Thanks @hlindberg!

**Pull request merged**